public/css/stylesheet.css

```diff
@@ -1214,10 +1214,6 @@ video {
   resize: both;
 }
 
-.list-inside {
```

Comment:
This style comes from tailwind. I believe since it doesn't see the style used in any classes it elides it from the compiled